tools/update_python_pip_packages.rb

```diff
@@ -6,47 +6,57 @@
 # Usage in root of cloned chromebrew repo:
 # tools/update_python_pip_packages.rb
 
-# Add >LOCAL< lib to LOAD_PATH
-$LOAD_PATH.unshift '../lib'
+# Add >LOCAL< lib to LOAD_PATH so that packages can be loaded
+$LOAD_PATH.unshift './lib'
+
 require_relative '../lib/color'
 require_relative '../lib/const'
+require_relative '../lib/package'
+require_relative '../lib/package_utils'
 require_relative '../lib/require_gem'
-require_gem('activesupport', 'active_support/core_ext/object/blank')
 require_gem 'concurrent-ruby'
+require_gem 'ruby-libversion', 'ruby_libversion'
```

Comment:
Why do we need 2 different gems?

Reply:
We need to install the gem ruby-libversion, and then require it as ruby_libversion